### DISCUSSION TITLE
W10 clients can not access remote, all on LAN

### DISCUSSION THREAD

**Discussion by mov02pc** [rustdesk/rustdesk#5956] [General]
### Bug Description

Hi there,

Description of the setup:
Server: Windows 10, RustDesk server pro version 1.1.10-3, also RustDesk client 1.2.2
Clients: several Windows 10 client 1.2.2 and iOS devices.
All device are on the same LAN, some wired some wireless.
Clients can successfully access each other, also iOS devices can access all clients,
But neither client can access the Server, and fails with “Connection error, Deadline has elapsed”, both W10 and iOS.
Attempt to access any W10 client from the Server throws “Connection error, Reset by the peer”

Strange is it used to work OK, but after rebooting the issue appeared.
Reinstalling client on the Server did not help.

Any hints or ideas what can be the reason?
Any log files to check?

Thanks,
 

### How to Reproduce

Have Server and Client installed on a Windows 10 host, and a client, either W10 or iOS.

### Expected Behavior

Expecting client to access the Server.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.2 ->1.2.2

### Screenshots

![IMG_0023](https://github.com/rustdesk/rustdesk/assets/147330408/f4cf5fdd-1483-4fed-9492-9893b34e72ad)
![IMG_0022](https://github.com/rustdesk/rustdesk/assets/147330408/4e2fa745-5e28-411d-8d69-c07d75b8e66a)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet
https://github.com/rustdesk/rustdesk/wiki/FAQ#force-relay
A better place for solving networking (lan/wlan, subnet etc) is on our discord, https://discord.gg/nDceKgxnkV
